helios-services/src/main/java/com/spotify/helios/master/ZooKeeperMasterModel.java

```diff
@@ -350,6 +350,8 @@ public void addDeploymentGroup(final DeploymentGroup deploymentGroup)
             create(Paths.configDeploymentGroup(deploymentGroup.getName()), deploymentGroup),
             create(Paths.statusDeploymentGroup(deploymentGroup.getName())),
             create(Paths.statusDeploymentGroupHosts(deploymentGroup.getName()),
+                   Json.asBytesUnchecked(emptyList())),
+            create(Paths.statusDeploymentGroupRemovedHosts(deploymentGroup.getName()),
```

Comment:
The best approach is for every piece of code that tries to check this path to be able to handle the case where it does not exist.